### ISSUE TITLE
Port isn't integer?

### ISSUE THREAD

**Issue by withinboredom** [gmr/consulate#45]
Error from cli:

```
consulate register -p 3306 mysql-master no-check
Traceback (most recent call last):
  File "/usr/local/bin/consulate", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/consulate/cli.py", line 283, in main
    register(consul, args)
  File "/usr/local/lib/python2.7/dist-packages/consulate/cli.py", line 255, in register
    args.port, tags, check, interval, ttl)
  File "/usr/local/lib/python2.7/dist-packages/consulate/api/agent.py", line 181, in register
    raise ValueError('port must be an integer')
ValueError: port must be an integer
```

**Comment by gmr** [comment]
That's pretty odd, since it should be forced to int by ``argparse`` I am forcing this to an int in CLI in an incoming patch, but I'm worried about TTL and other settings as well.  What version of Python 2.7 are you running?

**Comment by gmr** [comment]
This will be released with 0.6.0

**Comment by withinboredom** [comment]
I was running whatever the default was in the official ubuntu docker image.
